src/GameLogic/GameSession.ts

```diff
@@ -51,4 +50,14 @@ export class GameSession {
 		 */
 		this.turnRunner.runTurn(playerInput, this.level);
 	}
+
+	private _addProtagonist(isPlayerControlled: boolean, movesQueue: ActionSequence = new ActionSequence()): void {
```

Comment:
I've never done things this way, where the private methods are prefixed with an underscore - only for private variables. If you'd like to do it we'd have to change all the existing private variables to follow suit.

Reply:
The linter made me do it